Manual_for_achitecture_browser/index.asciidoc

```diff
@@ -0,0 +1,76 @@
+= Manual for architecture browser
+These content help you to know how you can contribute to the content of the architecture browser .
+
+== Stack specific solution
+A stack specific solution is the place where you can find all the solutions of each stack. 
+You should do each solution on a new branch and not take the master branch because that can lead to conflicts.
+
+== Fork repository
+First of all, you have to fork your own copy of devonfw/architectures to your account, by clicking on the icon "Fork".
+
+image::fork.PNG[Fork_Repository,100,100]
+
+== Open the repository on Gitpod
+The repository is already on your account. You have to click on "code" and copy the url.
```

Comment:
You can also add a link to https://gitpod.io/from-referrer/ to the readme file: https://spectrum.chat/gitpod/general/how-to-specify-users-fork-instead-of-main-repository~d4c0e615-511d-409e-9ffc-796869fe4708?m=MTU2NTI0NTAzNTI2OA==